scripts/aws/setupdb.sh

```diff
@@ -14,3 +14,14 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 
 # Run migrations
 envdir /etc/mmw.d/env /opt/app/manage.py migrate
+
+# Fetch boundary layer sql files
+FILE_HOST="http://santacruz.internal.azavea.com:9999"
+curl $FILE_HOST/boundary_huc08.sql.tar.gz | tar xz
+curl $FILE_HOST/boundary_huc10.sql.tar.gz | tar xz
+curl $FILE_HOST/boundary_huc12.sql.tar.gz | tar xz
+
+psql -f boundary_huc08.sql
```

Comment:
This is the type of command I was thinking:

```bash
$ curl -s $FILE_HOST/boundary_huc08_data.tar.gz | gunzip | psql -f boundary_huc08.sql
```

Let me know when you get the two separated and I can help with testing.